frontends/ol-search-ui/src/components/ExpandedLearningResourceDisplay.tsx

```diff
@@ -30,6 +32,40 @@ type Props = {
   runId: string | number | undefined
   setShowResourceDrawer: (params: ResourceIdentifiers | null) => void
 }
+export const formatDurationClockTime = (value: string) => {
```

Comment:
**Requested Change:** Could you move the comment to a JSDoc docstring:. Something like

```ts
/**
 * Format an ISO-8601 duration string so to a readable format
 * The logic here ensures that if there is a colon (:) to the left
 * of a time component (minutes, seconds) it is zero-padded
 * hours are not included if they are zero
 * this follows what most humans would consider a reasonable "clock display" format
 *
 * Example return values:
 *  - 3:00:01
 *  - 43:07
 *  - 3:09
 *  - 0:47
 */
export const formatDurationClockTime = (value: string): string => { /* */ }
```
Then it shows up on hover:
<img width="300" alt="Screen Shot 2022-09-08 at 10 01 17 AM" src="https://user-images.githubusercontent.com/9010790/189142107-622d16f8-cda2-4a00-b781-773560e6a7e3.png">

**Other Suggestions:**
- I would be inclined to extract it to `ol-utils`. Not necessarily for reuse, but because it is a self contained chunk of code that makes perfect sense in isolation.
- IMO, it would also be worth writing a few simple tests for this.
- Use `padStart` instead of ``0${duration.minutes().toString()}`.slice(-2)`:
    - the lodash [padStart](https://lodash.com/docs/4.17.15#padStart) function
    - or the [builtin String.prototype.padStart](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart), though [caniuse](https://caniuse.com/?search=padstart). says its browser support is only 95%